src/ol_infrastructure/applications/ocw_site/__main__.py

```diff
@@ -455,6 +464,15 @@
                 type="error",
             ),
         ],
+        logging_s3s=[
+            fastly.ServiceVclLoggingS3Args(
+                bucket_name=fastly_access_logging_bucket["bucket_name"],
+                name=f"ocw-{purpose}-{stack_info.env_suffix}-logging-args",
+                path=f"/ocw/{purpose}/{stack_info.env_suffix}/",
```

Comment:
Where is the `purpose` coming from?

Reply:
line 260 02b4e324687890ce84c5ac4b69ecca040da35c50c91f9ba8e9c290ee6c21774cR260

"live" or "draft", a distinction only important for OCW I believe.

Reply:
I'd say probably put `env_suffix` as the second tier rather than 3rd